lib/src/main/java/org/team100/lib/localization/PoseEstimationHelper.java

```diff
@@ -27,17 +27,20 @@ public class PoseEstimationHelper {
     private static final Telemetry t = Telemetry.get();
     private static final String m_name = Names.name(PoseEstimationHelper.class);
 
-    public static Translation2d cameraRotationToRobotRelative(Transform3d cameraInRobotCoordinates, Rotation3d cameraObject) {
+    public static Translation2d cameraRotationToRobotRelative(Transform3d cameraInRobotCoordinates, Rotation3d yawPitch) {
```

Comment:
i don't understand what this method is trying to do at all.  add a long comment explaining it.